IOHomework/src/main/java/sales/products/SalesStatistics.java

```diff
@@ -0,0 +1,88 @@
+package sales.products;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SalesStatistics {
+    public SalesStatistics() {
+    }
+
+    /**
+     * This method gets number of units sold for each product
+     * Create a map with key: productName and a counter which increments when a sale was done
+     */
+
+    public Map<String, Integer> sortByNumberOfProducts(List<Sales> sales) {
```

Comment:
With the current design the sort methods could be static. Your class has no state!